notebook/serverextensions.py

```diff
@@ -106,20 +106,21 @@ def validate_serverextension(import_name, logger=None):
         version = getattr(mod, '__version__', '')
     except Exception:
         logger.warning("Error loading server extension %s", import_name)
-
-    import_msg = u"     {} is {} importable?"
-    if func is not None:
-        infos.append(import_msg.format(GREEN_OK, import_name))
-    else:
-        warnings.append(import_msg.format(RED_X, import_name))
-
-    post_mortem = u"      {} {} {}"
-    if logger:
-        if warnings:
-            [logger.info(info) for info in infos]
-            [logger.warn(warning) for warning in warnings]
+        sys.exit(1)
```

Comment:
We shouldn't exit the process since this method is called multiple times from [`ListServerExtensionsApp`](https://github.com/jupyter/notebook/blob/master/notebook/serverextensions.py#L240) and an early warning will prevent listing subsequent extensions.